src/test/java/api/support/builders/DeclareItemLostRequestBuilder.java

```diff
@@ -0,0 +1,73 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.UUID;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+
+public class DeclareItemLostRequestBuilder extends JsonBuilder implements Builder {
+  private final UUID loanId;
+  private final DateTime dateTime;
+  private final String comment;
+  private final int expectedResponseStatusCode;
```

Comment:
I don’t think a request representation builder should know what response an potential invocation should give. These seem like different responsibilities to me. 

How can it be known at this point what the response will always be? It could be based upon other state of the system.

Reply:
Done.